docs/dev-guide/web-sdk.md

```diff
@@ -0,0 +1,74 @@
+---
+id: dev-guide-web-sdk
+title: Web SDK
+---
+
+The Jitsi Meet Web SDK provides the same user experience as the Jitsi Meet app, in a customizable way which you can embed in your apps.
+
+## Sample applications using the SDK
+If you want to see how easy integrating the Jitsi Meet Web SDK into a web application is, take a look at the [sample applications directory](https://github.com/jitsi/jitsi-meet-web-sdk/tree/main/examples).
+
+## Installation
+To access the Web SDK modules in your application you need to install it as a dependency:
+```bash
+npm install @jitsi/web-sdk
+```
+
+## Modules
+
+### fetchExternalApi
+To import the Jitsi Meet External API in your application, regardless of any frameworks or libraries that the application might depend on:
+```javascript
+window.onload = () => {
+    fetchExternalApi().then(JitsiMeetExternalApi => {
+        const api = new JitsiMeetExternalApi("YOUR_DOMAIN", options);
+    });
+}
+```
+More on the options parameter can be read [here](dev-guide/iframe.md#creating-the-jitsi-meet-api-object).
+
+### JitsiMeeting
+:::important
+React 16 or higher is required.
+:::
+To be used with custom domains as-it-is in React projects:
+```jsx
+<JitsiMeeting
```

Comment:
Add some config overrides and some PleaseUseAGoodRoomName in the example here so it feels more complete.